{
  "@class" : "org.sehkah.ddon.tools.extractor.common.logic.resource.entity.quest.QuestTextDataList",
  "FileHeader" : {
    "MagicBytesLength" : 4,
    "MagicString" : "QTD\u0000",
    "VersionBytesLength" : 4,
    "VersionNumber" : 2
  },
  "FileSize" : 52,
  "QuestTextDataList" : [ {
    "@class" : "org.sehkah.ddon.tools.extractor.common.logic.resource.entity.quest.QuestTextData",
    "Message" : "A Sudden Urge to Kill",
    "MsgGmdIdx" : 0,
    "Type" : 0,
    "TypeName" : "QUEST_TEXT_TYPE_NAME"
  }, {
    "@class" : "org.sehkah.ddon.tools.extractor.common.logic.resource.entity.quest.QuestTextData",
    "Message" : "Defeat the enemy encountered",
    "MsgGmdIdx" : 1,
    "Type" : 1,
    "TypeName" : "QUEST_TEXT_TYPE_ORDER"
  }, {
    "@class" : "org.sehkah.ddon.tools.extractor.common.logic.resource.entity.quest.QuestTextData",
    "Message" : "<NAME AREA>からの報告によると\r\n魔物の勢力が一時的に増し、危険な状況にあるという",
    "MsgGmdIdx" : 2,
    "Type" : 3,
    "TypeName" : "QUEST_TEXT_TYPE_FIND_INFO"
  }, {
    "@class" : "org.sehkah.ddon.tools.extractor.common.logic.resource.entity.quest.QuestTextData",
    "Message" : "なんとも恥ずべき失態だが\r\n俺たちは、敵にまんまと一杯喰わされちまったんだ\r\n仲間は傷を負い\r\nほうほうの体で逃げ出すしかなかったよ\r\n仇を頼みたい、まだ敵はあのあたりにいるはずだ",
    "MsgGmdIdx" : 3,
    "Type" : 4,
    "TypeName" : "QUEST_TEXT_TYPE_FIND_INFO_DETAIL"
  }, {
    "@class" : "org.sehkah.ddon.tools.extractor.common.logic.resource.entity.quest.QuestTextData",
    "Message" : "Defeat the enemy encountered",
    "MsgGmdIdx" : 4,
    "Type" : 2,
    "TypeName" : "QUEST_TEXT_TYPE_PURPOSE"
  } ]
}